src/main/scala/xerial/sbt/Sonatype.scala

```diff
@@ -45,6 +44,15 @@ object Sonatype extends sbt.Plugin {
     val sonatypeDefaultResolver = settingKey[Resolver]("Default Sonatype Resolver")
   }
 
+  object autoImport {
+    val SonatypeKeys = Sonatype.SonatypeKeys
```

Comment:
Autoplugins have their autoPlugin property auto imported (not everything in the plugin).  This ensures that SonatypeKeys is available, as before.